tests/install_upgrade_operators/product_install/conftest.py

```diff
@@ -298,17 +298,19 @@ def installed_hpp(cluster_backend_storage, hpp_volume_size):
 
 
 @pytest.fixture(scope="session")
-def cnv_version_to_install(is_production_source, openshift_current_version, cnv_image_url):
+def cnv_version_to_install_info(is_production_source, ocp_current_version, cnv_image_url):
     latest_z_stream = None
```

Comment:
though not from this PR but no need to set `latest_z_stream`; it is already set in both `if` and `else` clauses.